client/src/modules/stock/adjustment/adjustment.js

```diff
@@ -26,6 +26,7 @@ function StockAdjustmentController(
   // global variables
   vm.Stock = new StockForm('StockAdjustment');
   vm.movement = {};
+  vm.ROW_ERROR_FLAG = bhConstants.grid;
```

Comment:
Same comment - this should be `vm.ROW_ERROR_FLAG = bhConstants.grid.ROW_ERROR_FLAG`.